tests/installation/bootloader_uefi.pm

```diff
@@ -44,11 +44,18 @@ use bootloader_setup;
 use registration;
 use utils;
 use version_utils qw(is_jeos is_microos is_opensuse is_sle is_selfinstall);
+use Utils::Backends qw(is_ipmi);
 
 # hint: press shift-f10 trice for highest debug level
 sub run {
     my ($self) = @_;
 
+    # Press key 't' to let grub2 boot menu show up in serial console
+    if (is_ipmi && is_selfinstall && get_var('IPXE_UEFI')) {
```

Comment:
This if 
```
if (is_ipmi && is_selfinstall && get_var('IPXE_UEFI'))
```
is not good enough. You should add something like is_serial_console in it.

Reply:
Actually now only this way is supported, and we know it is sol and needs typing `t`. So no need for such check. If you insist, I can add one more condition.

I had a check in testapi. `is_serial_console ` is not proper. But `current_console` works to tell if sol console is used.  Ok for you?

Reply:
current_console is OK.

Reply:
done